js/apps/keycloak-server/README.md

```diff
@@ -41,6 +41,16 @@ If you want to run with a local Quarkus distribution of Keycloak for development
 pnpm start --local
 ```
 
+If you want to run with `IDELauncher`, make sure you set the following environment variables.
```

Comment:
This is not specific to `IDELauncher`, but pertains to all instances where a Keycloak server is started.
```suggestion
If you want to run Keycloak standalone (without the script) against the Vite development server, you can set the following environment variables to achieve the same result:
```

Reply:
make sense to me